patches/minecraft/net/minecraft/client/model/EntityModel.java.patch

```diff
@@ -0,0 +1,22 @@
+--- a/net/minecraft/client/model/EntityModel.java
++++ b/net/minecraft/client/model/EntityModel.java
+@@ -8,7 +_,8 @@
+ import net.minecraftforge.api.distmarker.OnlyIn;
+ 
+ @OnlyIn(Dist.CLIENT)
+-public abstract class EntityModel<T extends Entity> extends Model {
++public abstract class EntityModel<T extends Entity> extends Model implements net.minecraftforge.client.animation.IAnimatedModel
++{
```

Comment:
Please put the brace back on the same line.

Reply:
Fixed